docs/pages/jpsreport/jpsreport_inifile.md

```diff
@@ -0,0 +1,373 @@
+
+Possible parameters are:
+  - `measurement_area` given by an `id` number.
+     Note that the measurement area for method_B should be
+     rectangle based on the definition of the method.
+
+Possible output data are:
+  - `/Fundamental_Diagram/TinTout/`: output file `FDTinTout_traj_` with mean density and velocity of each pedestrian (PersID, $$\rho_i$$ and $$v_i$$).
+
+
+### Method C
+For definition see [Method C](jpsreport_method_C.html). Method C is used to analyze the steady state.
+
+```xml
+  <method_C enabled="true">
+      <measurement_area id="1"/>
+  </method_C>
+```
+
+Possible parameters are:
+
+- `id` indicates the size and location of the measurement_area.
+   Several measurement areas can be set in one inifile.
+
+- up to version 0.8.5:  `plot_time_series` specifies whether output the ($$\rho-t$$) and ($$v-t$$) diagrams.
+
+
+Possible output data are:
+  - `/Fundamental_Diagram/Classical_Voronoi/`: output file `rho_v_Classic_traj_` with mean density and velocity of over time (frame, $$rho(t)$$, $$v(t)$$).
+
+
+### Method D
+For definition see [Method D](jpsreport_method_D.html). Method D is used to analyze the steady state.
+
+```xml
+ <method_D enabled="true">
+   <measurement_area id="1" start_frame="None" stop_frame="None"
+         get_individual_FD="false"/>
+   <one_dimensional enabled="false"/>
+   <cut_by_circle enabled="false" radius="1.0" edges="10"/>
+   <profiles enabled="false" grid_size_x="0.20" grid_size_y="0.20"/>
+ </method_D>
+```
+
+Possible parameters are:
+
+- For each `measurement_area`, several id numbers can be set in one inifile.
+  `start_frame` and `stop_frame` give the starting and ending frame for data analysis.
+  The default values of these two parameters are `None`.
+  If you plan to analysis the whole run from beginning to the end,
+  set both of `start_frame` and `stop_frame` as `None`;
+  If `start_frame =None` but `stop_frame` is not,
+  then analysis will be performed from beginning of the trajectory to the `stop_frame`.
+  If `start_frame` is not `None` but `stop_frame = None`,
+  it will analyze from the `start_frame` to the end of the movement.
+  `get_individual_FD` determines whether or not to output the data
+  for individual fundamental diagram in the given measurement area,
+  which is based on the Voronoi density $$\rho_i$$,  velocity $$v_i$$, position ($$x_i$$,$$y_i$$ and $$z_i$$) and Voronoi polygon
+  of each pedestrian $$i$$ in a given measurement area but not mean
+  value over space.
+  If true, the related data will be written in the
+  folder `./Output/Fundamental_Diagram/IndividualFD/` in the output file `IFD_D_`.
+
+ - up to version 0.8.5: `plot_time_series` specifies whether output the $$\rho-t$$ and $$v-t$$-diagram.
+
+- `one_dimensional` should be used when pedestrians move on a line
```

Comment:
This could also be a separate tutorial.

Reply:
Yes, I think so. 
Actually, you have a lot of notebooks 😁 
if you convert them to md then this would be very very nice.

```
jupyter nbconvert --to markdown HowTo.ipynb
```

For example: 
http://www.chraibi.de/jekyll/jpsreport_howto.html

Reply:
A good idea. ;-)